x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/common/validation/SourceDestValidatorTests.java

```diff
@@ -464,21 +478,11 @@ public void testCheck_GivenDestIndexIsAliasThatMatchesMultipleIndices() throws I
                 CLUSTER_STATE,
                 new String[] { SOURCE_1 },
                 DEST_ALIAS,
-                SourceDestValidator.NON_DEFERABLE_VALIDATIONS,
+                Collections.emptyList(),
```

Comment:
not getting this change, should the test be removed?

Reply:
These second assertions in those tests seem to be testing that when the validation is not present then the error does not occur. Previously the `NON_DEFERABLE_VALIDATIONS` set was passed around but I could not see what the difference was compared to passing an empty list. Does this make sense?